tests/mpd/protocol/test_stored_playlists.py

```diff
@@ -211,29 +211,95 @@ def test_load_unknown_playlist_acks(self):
         self.assertEqualResponse('ACK [50@0] {load} No such playlist')
 
     def test_playlistadd(self):
+        tracks = [
+            Track(uri='dummy:a'),
+            Track(uri='dummy:b'),
+        ]
+        self.backend.library.dummy_library = tracks
+        self.backend.playlists.set_dummy_playlists([
+            Playlist(
+                name='name', uri='dummy:a1', tracks=[tracks[0]])])
+        self.send_request('playlistadd "name" "dummy:b"')
+        self.assertInResponse('OK')
+        self.assertEqual(
+            2, len(self.backend.playlists.get_items("dummy:a1").get()))
```

Comment:
It can be beneficial to split the body of the test in three sections with an empty line inbetween. The three sections are given-when-then: the setup, the action, and the asserts. Here I'd add an empty line after the dummy backend setup and after the `send_request()` line.